integrations/GA4/ECommerceEventConfig.js

```diff
@@ -0,0 +1,86 @@
+const eventNamesConfigArray = [
+  // Browsing Section
+  {
+    src: ["products searched", "product searched"],
+    dest: "search",
+  },
+  { src: ["product list viewed"], dest: "view_item_list" },
+  // { src: ["product list filtered"], dest: "" },
+
+  // Promotion Section
+  { src: ["promotion viewed"], dest: "view_promotion" },
+  { src: ["promotion clicked"], dest: "select_promotion" },
+
+  // Ordering Section
+  {
+    src: ["product clicked", "products clicked"],
+    dest: "select_item",
+    hasItem: true,
+  },
+  { src: ["product viewed"], dest: "view_item", hasItem: true },
+  { src: ["product added"], dest: "add_to_cart", hasItem: true },
+  { src: ["product removed"], dest: "remove_from_cart", hasItem: true },
+  { src: ["cart viewed"], dest: "view_cart", hasItem: true },
+  { src: ["checkout started"], dest: "begin_checkout" },
+  { src: ["checkout step viewed"], dest: "" },
+  { src: ["checkout step completed"], dest: "" },
+  { src: ["payment info entered"], dest: "add_payment_info" },
+  { src: ["order updated"], dest: "" },
+  { src: ["order completed"], dest: "purchase" },
+  // { src: ["order refunded"], dest: "refund" }, GA4 refund is different it supports two refund, partial and full refund
+  { src: ["order cancelled"], dest: "" },
+
+  // Coupon Section
+
+  //----------
+  // do I need the two below events
+  // Wishlist Section
+  // { src: ["product added to wishlist"], dest: "add_to_wishlist" },
+  //-------
+
+  // Sharing Section
+  // { src: ["product shared", "cart shared"], dest: "share" },
+  //---------
+
+  // Reviewing Section
+
+  // --------
+];
+
+const eventParametersConfigArray = [
+  { src: "query", dest: ["search_term"] },
+  { src: "list_id", dest: ["item_list_id", "items.item_list_id"] },
```

Comment:
instead of array, use a flag like **storeAlsoInItems: true/false** 
that will make the parsing simpler?

Reply:
okkk looking into it